src/caustics/cosmology.py

```diff
@@ -43,12 +43,10 @@ class Cosmology(Parametrized):
     This class provides an interface for cosmological computations used in lensing
     such as comoving distance and critical surface density.
 
-    Units
-    -----
     Distance
-        Mpc
+        *Unit: Mpc*
```

Comment:
Rather than using `Mpc` as your unit, is there a slightly more descriptive name that you can give? For example, using `micrometers` instead of `mcm` gives the end user a better understand of what the units should be. Remember that these units will be rendered in the JupyterBook. So, contributors and users will use these units to understand the software. You can also use the first occurrence of this variable to write it out in more detail and use the abbreviations afterward.